src/components/Home/Aditya/Week3/components/counter/Counter.tsx

```diff
@@ -0,0 +1,31 @@
+import React, { useState } from 'react';
+import { CounterDiv } from './counterStyles';
+
+const Counter: React.FC = () => {
+  const [count, setCount] = useState<number>(0);
+  const handleChangeCounter = (e: React.MouseEvent<HTMLButtonElement>):void => {
+    setCount(count + (+e.currentTarget.innerText));
+  };
+
+  const changeOptions:number[] = [-1, 1, -2, 2, -5, 5, -10, 10];
+
+  return (
+    <CounterDiv>
```

Comment:
I recommend, CounterWrapper instead of CounterDiv

Reply:
@aditya-s-19  check at all places